xija/component/heat.py

```diff
@@ -51,6 +51,28 @@ class ActiveHeatPower(ModelComponent):
     pass
 
 
+class SolarHeatBase(PrecomputedHeatPower):
+
+    _t_phase = None
+
+    @property
+    def t_phase(self):
```

Comment:
This needs a deleter since there is a `delattr(self, 't_phase')` in the `epoch` setter.  (Don't know why I didn't write that as `del self.t_phase`, but no mind.).